tests/test_urls.py

```diff
@@ -16,9 +17,15 @@
     '404rst',
 )
 
+@pytest.fixture(autouse=True, scope='function')
+def remove_sphinx_build_output():
+    """Remove _build/ folder, if exist."""
+    for path in (srcdir, rstsrcdir):
+        build_path = os.path.join(path, '_build')
+        if os.path.exists(build_path):
+            shutil.rmtree(build_path)
+
```

Comment:
These should really go in a sphinx testing util library. Might be a good excuse to start one.

Reply:
sphinx itself have a util testing library, but I didn't find a method like this:

1. I didn't find enough, or
1. it does not exist and would be good to propose the fix upstream

See https://github.com/sphinx-doc/sphinx/blob/master/sphinx/testing/fixtures.py